staging/src/k8s.io/apiserver/pkg/storage/etcd/api_object_versioner_test.go

```diff
@@ -40,6 +41,41 @@ func TestObjectVersioner(t *testing.T) {
 	}
 }
 
+func TestEtcdParseWatchResourceVersion(t *testing.T) {
```

Comment:
I can see that this test for ParseWatchResourceVersion is moved from storage/util_test.go to storage/etcd/api_object_versioner_test.go and realize that, although there are Parse**Watch**ResourceVersion and Parse**List**ResourceVersion, there is no test for ParseListResourceVersion. My question is, do we need a test for Parse**List**ResourceVersion, or is there any other test already address that?

Reply:
I made the test test both.